### ISSUE TITLE
error in axis label with a simple latex expression

### ISSUE THREAD

**Issue by fabricix** [veusz/veusz#471]
The latex expression `\ddot{u}` is not working in axis label, while `\dot{u}` work fine:
![image](https://user-images.githubusercontent.com/22794916/99285511-61cabc00-2816-11eb-95da-d7677c347978.png)

The expected behavior of `\ddot{u}` is ![formula](https://render.githubusercontent.com/render/math?math=\ddot{u})

Program version: 3.0.1 and 3.2.1

**Comment by rodrigo-j-goncalves** [comment]
I can confirm this behavior in Veusz 3.2.1

In the [examples page](https://veusz.github.io/examples/) you can see (currently at the bottom of the page) that Veusz supports MathML (by the way, the [link](https://www.mathjax.org/demos/mathml-samples/) on that example is broken)

So, searching a bit in internet I [found](https://stackoverflow.com/questions/58755256/problems-when-embedding-latex-dot-and-hat-into-microsoft-word-via-mathml) this should do the trick (just copy all the code below  into the "label" textbox.

This is the code:
```
<math xmlns="http://www.w3.org/1998/Math/MathML" display="block">
  <mrow class="MJX-TeXAtom-ORD">
    <mi>u&#x0307;</mi>
  </mrow>
</math>
```
And this is how it will look like (bold is mine): **<math xmlns="http://www.w3.org/1998/Math/MathML" display="block">  <mrow class="MJX-TeXAtom-ORD"> <mi>u&#x0307;</mi> </mrow></math>**

The crucial part here is **&#x0307** placed after the letter. You may want to search for other symbols (probably not the right word, sorry).

**Comment by jeremysanders** [comment]
Thanks for the report. This should be fixed in the next release (>3.3.1).
